.lighthouse/jenkins-x/triggers.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: config.lighthouse.jenkins-x.io/v1alpha1
+kind: TriggerConfig
+spec:
+  presubmits:
```

Comment:
Also add a lint step, see jx-pipeline for example: https://github.com/jenkins-x-plugins/jx-pipeline/blob/main/.lighthouse/jenkins-x/triggers.yaml#L12